contracts/mixnet/src/storage.rs

```diff
@@ -315,10 +315,18 @@ pub fn reverse_mix_delegations_read<'a>(
     ReadonlyBucket::multilevel(storage, &[PREFIX_REVERSE_MIX_DELEGATION, owner.as_bytes()])
 }
 
+// This function will need to be deleted, once the migration is completed
```

Comment:
To not pollute the storage with migration specific functions, you could move it to the "migrate" function scope

Reply:
Moved them inside that scope, it looks much "dirtier", but I guess it will be easier to remove.